src/webapp/CompositionRoot.ts

```diff
@@ -116,6 +125,9 @@ export async function getCompositionRoot(instance: Instance) {
         analytics: getExecute({
             sendPageView: new SendPageViewUseCase(analyticsRepository, configRepository),
```

Comment:
Here we could use `analyticsConfigRepository` instead of using `configRepository`.